wp-includes/mysql/class-mysql-lexer.php

```diff
@@ -0,0 +1,2752 @@
+		'UPGRADE'                                => self::UPGRADE_SYMBOL,
+		'USAGE'                                  => self::USAGE_SYMBOL,
+		'USE'                                    => self::USE_SYMBOL,
+		'USE_FRM'                                => self::USE_FRM_SYMBOL,
+		'USER'                                   => self::USER_SYMBOL,
+		'USER_RESOURCES'                         => self::USER_RESOURCES_SYMBOL,
+		'USING'                                  => self::USING_SYMBOL,
+		'UTC_DATE'                               => self::UTC_DATE_SYMBOL,
+		'UTC_TIME'                               => self::UTC_TIME_SYMBOL,
+		'UTC_TIMESTAMP'                          => self::UTC_TIMESTAMP_SYMBOL,
+		'VALIDATION'                             => self::VALIDATION_SYMBOL,
+		'VALUE'                                  => self::VALUE_SYMBOL,
+		'VALUES'                                 => self::VALUES_SYMBOL,
+		'VAR_POP'                                => self::VAR_POP_SYMBOL,
+		'VAR_SAMP'                               => self::VAR_SAMP_SYMBOL,
+		'VARBINARY'                              => self::VARBINARY_SYMBOL,
+		'VARCHAR'                                => self::VARCHAR_SYMBOL,
+		'VARCHARACTER'                           => self::VARCHARACTER_SYMBOL,
+		'VARIABLES'                              => self::VARIABLES_SYMBOL,
+		'VARIANCE'                               => self::VARIANCE_SYMBOL,
+		'VARYING'                                => self::VARYING_SYMBOL,
+		'VIEW'                                   => self::VIEW_SYMBOL,
+		'VIRTUAL'                                => self::VIRTUAL_SYMBOL,
+		'WAIT'                                   => self::WAIT_SYMBOL,
+		'WARNINGS'                               => self::WARNINGS_SYMBOL,
+		'WEEK'                                   => self::WEEK_SYMBOL,
+		'WEIGHT_STRING'                          => self::WEIGHT_STRING_SYMBOL,
+		'WHEN'                                   => self::WHEN_SYMBOL,
+		'WHERE'                                  => self::WHERE_SYMBOL,
+		'WHILE'                                  => self::WHILE_SYMBOL,
+		'WITH'                                   => self::WITH_SYMBOL,
+		'WITHOUT'                                => self::WITHOUT_SYMBOL,
+		'WORK'                                   => self::WORK_SYMBOL,
+		'WRAPPER'                                => self::WRAPPER_SYMBOL,
+		'WRITE'                                  => self::WRITE_SYMBOL,
+		'X509'                                   => self::X509_SYMBOL,
+		'XA'                                     => self::XA_SYMBOL,
+		'XID'                                    => self::XID_SYMBOL,
+		'XML'                                    => self::XML_SYMBOL,
+		'XOR'                                    => self::XOR_SYMBOL,
+		'YEAR'                                   => self::YEAR_SYMBOL,
+		'YEAR_MONTH'                             => self::YEAR_MONTH_SYMBOL,
+		'ZEROFILL'                               => self::ZEROFILL_SYMBOL,
+
+		// Tokens from MySQL 8.0:
```

Comment:
This is such a helpful distinction, thank you for documenting it like this :heart: